library/src/main/java/com/evernote/android/state/StateSaver.java

```diff
@@ -32,16 +32,19 @@
 
     private static final Map<Class<?>, Injector> INJECTORS = new LinkedHashMap<>();
 
-    private static Injector getInjector(Class<?> cls)
+    @Nullable
+    private static Injector getInjector(@NonNull Class<?> cls)
             throws IllegalAccessException, InstantiationException {
-        Injector injector = INJECTORS.get(cls);
-        if (injector != null || INJECTORS.containsKey(cls)) {
```

Comment:
This was a small optimization. `INJECTORS` could contain a null value, so the `containsKey` check is necessary, see #31

Reply:
I know, but NPE check is duplicated in `safeGet` method.
So I don't think it's necessary.

Reply:
It is, because then it saves the expensive reflection call.